src/components/DataTable/DataTable.vue

```diff
@@ -13,7 +13,7 @@
         <template v-for="(column, columnIndex) in columns" :key="columnIndex">
           <th
             class="data-table__table-header"
-            :style="{ textAlign: column.align ?? 'left' }"
+            :style="{ textAlign: column.align ?? 'left', cursor: column.sortable ?? 'pointer' }"
```

Comment:
Is this right? It seems like it's pointer if sortable is nullish. But I think we want pointer if sortable is true? Maybe I've got it backwards in my head

Reply:
Yes, you're right we should show the pointer if it's true. Since, I made `sortable` optional in `DataTableColumn.ts`, it'll be undefined if not provided so for the change I'm thinking  `cursor: column.sortable || column.sortable == undefined ? 'pointer' : ''` but I feel like there's a better way to check that?

Reply:
so if sortable is optional then its default value will be `false`. So this should just be `cursor: column.sortable ? 'pointer': ''`. And then we update miter in the other projects we'll add `sortable` to the columns that need sorting. 

We'll be changing the default behavior but that's the correct behavior for a boolean prop.